src/main/java/frc/utils/ChargeStationStates.java

```diff
@@ -71,34 +75,34 @@ public int secondsToTicks(double time) {
     public double autoBalanceRoutine() {
         switch (state) {
             // drive forwards to approach station, exit when tilt is detected
-            case 0:
+            case APPROACH:
                 if (getTilt() > onChargeStationDegree) {
                     debounceCount++;
                 }
                 if (debounceCount > secondsToTicks(debounceTime)) {
-                    state = 1;
+                    state = States.CLIMBING_CHARGE_STATION;
                     debounceCount = 0;
                     return robotSpeedSlow;
                 }
                 return robotSpeedFast;
             // driving up charge station, drive slower, stopping when level
-            case 1:
+            case CLIMBING_CHARGE_STATION:
                 if (getTilt() < levelDegree) {
                     debounceCount++;
                 }
                 if (debounceCount > secondsToTicks(debounceTime)) {
```

Comment:
What are you debouncing here? Debounce is usually in reference to buttons.

Reply:
I used it to eliminate accelerometer fluctuations, just like button debouncing eliminates false positives on button presses due to small spring oscillations.